### DISCUSSION TITLE
Can i use Google Authenticator as a MFA factor?

### DISCUSSION THREAD

**Discussion by chaitanyapotti** [Web3Auth/Web3Auth#338] [Feature Requests]
Google authenticator is a very convenient solution to manage my multi-factor authentication setup.
I would like to use it with my web3auth account

**Comment by YZhenY** [comment]
Google Authenticator works similar to other FIDO WebAuthn stnadards, in that it uses a key generated on applications, and nonces + signatures on timestamps to create codes for specific applications.

As such, an attestation from Google Authenticator can't hold cryptographic material unless done via a proxy, making that factor potentially custodial

Whilst we can't use Google Authenticator in a non-custodial way out, there are alternatives on the mobile application level to just store device cryptographic material on that device - essentially achieving the same outcome of using ownership of a device to secure a users account. This is possible across the board on our SDKs
